docs/userstories/ClusterNetworkPolicy/Tenant_Namespace_Isolation.md

```diff
@@ -0,0 +1,28 @@
+### Summary
+
+As a cluster admin, I want to isolate all the tenants (modeled as Namespaces)
+on my cluster from each other by default.
+
```

Comment:
"Isolate all tenants" seems pretty specific here, should't this story just say we would like to apply network policy rules at a cluster scope rather than namespace scope?